src/bitpay/models/facade.py

```diff
@@ -1,6 +1,7 @@
 """
 Facade
 """
+
 from enum import Enum
 
 from bitpay.exceptions.bitpay_exception import BitPayException
```

Comment:
@mwarzybok-sumoheavy this was removed from the `5.0.x` version -- should we remove it here, too?

Reply:
yes :)